packages/react/select/src/Select.tsx

```diff
@@ -105,6 +106,7 @@ const Select: React.FC<SelectProps> = (props: ScopedProps<SelectProps>) => {
   // We set this to true by default so that events bubble to forms without JS (SSR)
   const isFormControl = trigger ? Boolean(trigger.closest('form')) : true;
   const [bubbleSelect, setBubbleSelect] = React.useState<HTMLSelectElement | null>(null);
+  const triggerPointerDownRef = React.useRef<{ x: number; y: number } | null>(null);
```

Comment:
Should I rename this `triggerPointerDownPosRef`?

Reply:
Renamed as I think it makes more sense.

Reply:
Yeah, totally